flexmeasures/data/schemas/tests/test_scheduling.py

```diff
@@ -215,3 +216,51 @@ def load_schema():
             load_schema()
     else:
         load_schema()
+
+
+@pytest.mark.parametrize(
+    ["flex_context", "fails"],
+    [
+        (
+            {"site-power-capacity": -1},
+            {"site-power-capacity": "Unsupported value type"},
+        ),
+        (
+            {"site-power-capacity": "-1 MVA"},
+            {"site-power-capacity": "Must be greater than or equal to 0."},
+        ),
+        (
+            {"site-power-capacity": "1 MVA"},
+            False,
+        ),
+        (
+            {"site-power-capacity": {"sensor": "site-power-capacity"}},
+            False,
+        ),
+    ],
+)
+def test_flex_context_schema(db, app, setup_site_capacity_sensor, flex_context, fails):
```

Comment:
Nice!

I wonder if we need another test to check that the values are properly set in on the `device_constraint`. For other fields, that's the case so we are already covering most cases of fetching the value from a quantity, sensor or fallback attribute.

Reply:
I expanded an existing test to cover the site capacities, too.